pkg/apis/cluster/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -128,6 +128,7 @@ func (in *ClusterNetworkingConfig) DeepCopy() *ClusterNetworkingConfig {
 func (in *ClusterSpec) DeepCopyInto(out *ClusterSpec) {
 	*out = *in
 	in.ClusterNetwork.DeepCopyInto(&out.ClusterNetwork)
+	in.ProviderConfig.DeepCopyInto(&out.ProviderConfig)
```

Comment:
Has no on run this since that was changed?

Reply:
don't think so